README.md

```diff
@@ -129,6 +129,26 @@ stackit config list
 
 You can also edit the configuration file manually.
 
+## Customization
+
+### Using **less** as a pager
+
+STACKIT CLI uses the `less` as default pager. 
+When using `less` as a pager, STACKIT CLI will automatically pass following options
+- -F, --quit-if-one-screen - Less will automatically exit if the entire file can be displayed on the first screen.
+- -S, --chop-long-lines - Lines longer than the screen width will be chopped rather than being folded.
+- -w, --hilite-unread - Temporarily highlights the first "new" line after a forward movement of a full page.
+
+
+>[IMPORTANT]
+> These options will not be added if:
+> - The pager is not named `less`.

```

Comment:
Can't the user pass in flags for their custom pager by setting `PAGER` to something like `less -x`?

Reply:
```suggestion
> These options will not be added automatically if a custom pager is defined.
```

Reply:
In the current state of implementation, no args can be passed. Should this be possible? 

If we want to support args, we need to split the env variable content as command and args.

Reply:
Even if the env variable is defined as `PAGER="less -x -y -z"`?

Reply:
yes, even if we pass args like that

Reply:
That's weird :thinking:

Reply:
I would say it's not a must have, but also the other CLIs don't support this?

Reply:
@vicentepinto98 the one I checked handles the given string differently to detect parameters and pass as args in the exec.command

Reply:
@joaopalet extended README for the env variables to define parameters as discussed tomorrow :)